app/src/main/java/com/orange/ouds/app/ui/utilities/composable/CustomizationBottomSheet.kt

```diff
@@ -94,7 +94,7 @@ fun CustomizationBottomSheetScaffold(
                     }
                     .fillMaxWidth()
                     .height(BottomSheetDefaults.SheetPeekHeight)
-                    .padding(horizontal = OudsTheme.spaces.fixed.medium),
+                    .padding(horizontal = OudsTheme.grids.margin),
```

Comment:
The grid margin should also be applied to the content of the sheet, which is not the case because we use `OudsTheme.spaces.fixed.medium` in `CustomizationElements.kt`.

BTW, if I am not wrong the grid margin is also missing in variants screen, component detail screen and about screen (although this one is temporary so I don't know if we should waste time on this).

Reply:
Sometimes we are using the native `ListItem` component in edge-to-edge so I didn't change anything for the moment in that case.